storybook/stories/TagLabel.stories.js

```diff
@@ -0,0 +1,51 @@
+import { storiesOf } from '@storybook/vue'
+import Tag from '@mesg-components/tag-label'
+
+storiesOf('Tag label', module)
+  .add('default', () => ({
+    components: { Tag },
+    template: `
+    <div class="container">
```

Comment:
please put just some texts in the tag, it doesn't make sense to have h1-2-3

Reply:
Fixed